tests/functional/tables/e2e_tables.py

```diff
@@ -123,3 +123,45 @@ def test_filter(self):
         self.assertEqual(self.vt.filter.index.tolist(), [112166, 112167])
         self.assertIn("14_52004545_C>CT", self.vt.filter.columns)
         self.assertEqual(self.vt.filter.loc[112166, "14_52004545_C>CT"], "PASS")
+
+
+class TestQCTables(BaseResdkFunctionalTest):
+    def setUp(self):
+        self.cache_dir = tempfile.mkdtemp()
+        self.test_server_url = TABLES_LIVE_URL
+        self.test_collection_slug = "qctables-test-collection"
+        self.res = resdk.Resolwe(
+            url=self.test_server_url,
+            username=TABLES_USER_EMAIL,
+            password=TABLES_USER_PASSWORD,
+        )
+        self.collection = self.res.collection.get(self.test_collection_slug)
+        self.qt = QCTables(self.collection, cache_dir=self.cache_dir)
+
+    def tearDown(self):
+        shutil.rmtree(self.cache_dir)
+
+    def test_rnaseq_qc(self):
+        self.assertEqual(
+            self.qt.fetch_group(group_name="rnaseq").loc[139805, "gc_content_raw"], 52.0
+        )
+
+    def test_chipseq_qc(self):
+        self.assertEqual(
+            self.qt.fetch_group(group_name="chipseq").loc[
+                139801, "control_prepeak_mapped_percentage"
+            ],
+            0.9018,
+        )
+
+    def test_wgs_qc(self):
```

Comment:
Would be nice to also test one of the property-like data type getters, e.g. `qt.general_alignment` or similar...